src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -6112,6 +6113,132 @@ void ObjectMgr::LoadQuestAreaTriggers()
+            if (!sObjectMgr->GetCreatureTemplate(id))
+            {
+                LOG_ERROR("sql.sql", "Table `quest_greeting`: creature template entry {} does not exist.", id);
+                continue;
+            }
+            break;
+        case 1: // GameObject
+            if (!sObjectMgr->GetGameObjectTemplate(id))
+            {
+                LOG_ERROR("sql.sql", "Table `quest_greeting`: gameobject template entry {} does not exist.", id);
+                continue;
+            }
+            break;
+        default:
+            continue;
+        }
+
+        uint16 greetEmoteType = fields[2].Get<uint16>();
+        uint32 greetEmoteDelay = fields[3].Get<uint32>();
+        std::string greeting = fields[4].Get<std::string>();
+
+        _questGreetingStore[type].emplace(std::piecewise_construct, std::forward_as_tuple(id), std::forward_as_tuple(greetEmoteType, greetEmoteDelay, std::move(greeting)));
+
+        ++count;
+    }
+    while (result->NextRow());
+
+    LOG_INFO("server.loading", ">> Loaded {} quest_greeting in {} ms", count, GetMSTimeDiffToNow(oldMSTime));
+}
+
+void ObjectMgr::LoadQuestGreetingsLocales()
+{
+    uint32 oldMSTime = getMSTime();
+
+    _questGreetingLocaleStore.clear();
+
+    //                                               0     1      2       3
+    QueryResult result = WorldDatabase.Query("SELECT ID, Type, Locale, Greeting FROM quest_greeting_locale");
+    if (!result)
+    {
+        LOG_INFO("server.loading", ">> Loaded 0 quest_greeting locales. DB table `quest_greeting_locale` is empty.");
+        return;
+    }
+
+    uint32 count = 0;
+
+    do
+    {
+        Field* fields = result->Fetch();
+
+        uint32 id = fields[0].Get<uint32>();
+        uint8 type = fields[1].Get<uint8>();
+        switch (type)
+        {
+        case 0: // Creature
+            if (!sObjectMgr->GetCreatureTemplate(id))
+            {
+                LOG_ERROR("sql.sql", "Table `quest_greeting_locale`: creature template entry {} does not exist.", id);
+                continue;
+            }
+            break;
+        case 1: // GameObject
+            if (!sObjectMgr->GetGameObjectTemplate(id))
+            {
+                LOG_ERROR("sql.sql", "Table `quest_greeting_locale`: gameobject template entry {} does not exist.", id);
+                continue;
+            }
+            break;
+        default:
+            break;
+        }
+
+        std::string localeName = fields[2].Get<std::string>();
+
+        LocaleConstant locale = GetLocaleByName(localeName);
+        if (locale == LOCALE_enUS)
+            continue;
+
+        QuestGreetingLocale& data = _questGreetingLocaleStore[MAKE_PAIR32(type, id)];
+        AddLocaleString(fields[3].Get<std::string>(), locale, data.Greeting);
+
+        ++count;
+    } while (result->NextRow());
+
+    LOG_INFO("server.loading", ">> Loaded {} quest greeting locale strings in {} ms", (uint32)_questGreetingLocaleStore.count(), GetMSTimeDiffToNow(oldMSTime));
```

Comment:
```suggestion
    LOG_INFO("server.loading", ">> Loaded {} quest greeting locale strings in {} ms", (uint32)_questGreetingLocaleStore.size(), GetMSTimeDiffToNow(oldMSTime));
```
I have made this fix locally to allow build to complete. I will let you know if locale data now is working!